src/lib/dev-environment/dev-environment-cli.ts

```diff
@@ -830,43 +830,61 @@ export function processVersionOption( value: unknown ): string {
 export function addDevEnvConfigurationOptions( command: Args ): Args {
 	// We leave the third parameter to undefined on some because the defaults are handled in preProcessInstanceData()
 	return command
-		.option( 'wordpress', 'Use a specific WordPress version', undefined, processVersionOption )
-		.option( [ 'u', 'mu-plugins' ], 'Use a specific mu-plugins changeset or local directory' )
+		.option(
+			'wordpress',
+			'Manage the version of WordPress. Accepts a string value for major versions (6.x). Defaults to the most recent version of WordPress.',
+			undefined,
+			processVersionOption
+		)
+		.option(
+			[ 'u', 'mu-plugins' ],
+			'Manage the source for VIP MU plugins. Accepts "demo" (default) for a read-only image of the staging branch, or a path to a built copy of VIP MU plugins on the local machine.'
+		)
 		.option(
 			'app-code',
-			'Use the application code from a local directory or use "demo" for VIP skeleton code'
+			'Manage the source for application code. Accepts "demo" (default) for a read-only image of WordPress skeleton code, or a path to a git cloned application repo on the local machine.'
```

Comment:
```suggestion
			'Manage the source for application code. Accepts "demo" (default) for a read-only image of WordPress VIP skeleton application code, or a path to a VIP formatted application repo on the local machine.'
```
A couple of things I'd suggest on this change:

- We should be clear that it's the VIP skeleton repo since that's a format and repo specific for WordPress on VIP
- The path to the local code doesn't have to be git cloned, but it does have to be in the same/similar format as a VIP customer application repo

Reply:
I appreciate both of these suggestions and will make it so!